site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -256,6 +257,29 @@ image::installation-guides/cloud/deployments-destroying.png[The deployment is be
 
 After the deployment is completely removed, it will have a "Destroyed" state on the [breadcrumb]#Deployments# tab. At this point you will no longer be charged for this deployment.
 
+=== Migrating a Deployment
+
+You can migrate your user data to a FusionAuth Cloud deployment or from FusionAuth Cloud to a self-hosted FusionAuth instance. 
+
+==== Migrating To FusionAuth Cloud
+
+To migrate from a self-hosted instance to FusionAuth Cloud:
+
+* You must be running PostgreSQL. If you are running MySQL, please migrate to a PostgreSQL database locally. Here's a link:https://fusionauth.io/community/forum/topic/985/migrating-from-mysql-to-postgresql[forum post on how to do this].
+* Export your FusionAuth database to a file.
+* Encrypt the export file with a symmetric solution such as gpg.
+* Purchase a FusionAuth Cloud deployment that meets your needs. Select the version of FusionAuth that is the same as your local installation.
```

Comment:
If you need to update your cloud version, the time to do that is after importing your existing data.  The account dashboard has an upgrade button for this purpose.